### PR TITLE
fixes a bunch of turfs being active roundstart

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
fixes a bunch of turfs being active roundstart

## Why It's Good For The Game
Through testing this dropped the active roundstart turfs from ~700 to ~30, which is a great performance gain
I probably need to go through the ruins again, got most of the big offenders
Alters the "Visualise Active Turfs" verb to now apply an overlay to those turfs, allowing them to be more easily seen in testing

## Images of changes
Not playerfacing map changes

## Testing
Prevented the ticker from firing and used visualise active turfs to look at what turfs were active or not
## Changelog
:cl:
fix: Atmos related lag reduced roundstart
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Can you add a CI check where if there's any ATs after the SSair init it fails?

File: code/modules/admin/verbs/debug.dm [hunk 1]
```diff
@@ -974,6 +974,8 @@ GLOBAL_PROTECT(AdminProcCallSpamPrevention)
 		if(!zlevel_turf_indexes["[T.z]"])
 			zlevel_turf_indexes["[T.z]"] = list()
 		zlevel_turf_indexes["[T.z]"] |= T
+		T.color = "red"
+		T.maptext = "ACT_TURF"
```
Comment (AffectedArc07) [inline code/modules/admin/verbs/debug.dm:978]: Bad idea, we use this on the live server.

I wouldn't mind a "Mark ATs in world (do not use on live)" question though

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/debug.dm (hunk 1 above)
Comment (S34NW) [reply code/modules/admin/verbs/debug.dm:978]: hide it under a different set of verbs so the admin monkeys can't use it

Comment (GDNgit) [reply code/modules/admin/verbs/debug.dm:978]: I can just handle this client side then

Comment (GDNgit) [reply code/modules/admin/verbs/debug.dm]: handled client side now

**Comment by GDNgit** [comment]
Z level 3 is now ignored due to init issues

**Review by AffectedArc07 — APPROVED** [review]

File: code/modules/admin/admin_verbs.dm [hunk 2]
```diff
@@ -296,6 +296,7 @@ GLOBAL_LIST_INIT(admin_verbs_maintainer, list(
 			verbs += /client/proc/ss_breakdown
 			verbs += /client/proc/show_gc_queues
 			verbs += /client/proc/toggle_mctabs
+			verbs += /client/proc/visualise_active_turfs
```
[the remark this answers is not in the document]
Comment (GDNgit) [reply code/modules/admin/admin_verbs.dm:299]: Screenshot 2023-09-19 214107

**Pull request merged**